zwave_js_server/model/value.py

```diff
@@ -9,9 +9,9 @@
 
 def value_id(node: "Node", event: dict) -> str:
     """Return ID of value."""
-    command_class = event.get("commandClass")
-    endpoint = event.get("endpoint")
-    property_key_name = event.get("propertyKeyName")
+    command_class = event["commandClass"]
+    endpoint = event.get("endpoint", "")
```

Comment:
Should we be explicit instead and have a default like `"no_endpoint"`?

Reply:
I think that's fine as the ID is not supposed to be human readable.

Reply:
Yeah, but we'll still read this when debugging, right? Having a double dash is easy to glance over or misinterpret, I'm thinking.

Reply:
I'm fine with some default value like "00" or "no_endpoint".
As far as I can see nodes always have an endpoint.

Reply:
@MartinHjelmare changed default value to 00 and resolved the merge conflict